src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Store/ContactDiaryStoreV1.swift

```diff
@@ -854,26 +888,74 @@ class ContactDiaryStoreV1: DiaryStoring, DiaryProviding {
 	}
 }
 
+// MARK: Creation
+
 extension ContactDiaryStoreV1 {
-	convenience init(fileName: String) {
+
+	static func make(filename: String) -> ContactDiaryStoreV1 {
```

Comment:
Can we turn the naming the other way around, so that we can still use `init(filename:)` as the initializer used in AppDelegate?

Reply:
Was my first attempt. Compiler says no "A non-failable initializer cannot delegate to failable initializer"
Do you know how to do it?

Reply:
Okay, I understand. Spontaneously I do not have a solution. But I quickly tried and realized on another note, that the `filename` does not seem to be used anymore.

Reply:
Why move this into an extension instead inside the MARKs section?

Reply:
I moved it to an extension, because all this stuff is in context of "convenience creation". It does not really belongs to the store. Just logic to have a simpler creation process.